### DISCUSSION TITLE
Suggestion: Ghost (Blogging Platform)

### DISCUSSION THREAD

**Discussion by m1ckyb** [tteck/Proxmox#551] [Ideas]
Ghost is a powerful app for new-media creators to publish, share, and grow a business around their content. It comes with modern tools to build a website, publish content, send newsletters & offer paid subscriptions to members.

Install: https://ghost.org/docs/install/ubuntu/

**Comment by ajtatum** [comment]
I'd love to see this added... with the option to install it with MySQL

**Comment by halpcomputar** [comment]
I haven't tried the one available, but I noticed it's available as a template under TurnKey
